docs/livemedia-creator.1

```diff
@@ -184,6 +185,10 @@ volume id
 \fB\-\-squashfs_args SQUASHFS_ARGS\fR
 additional squashfs args
 
+.TP
+\fB\-\-timeout TIMEOUT\fR
+Cancel installer after X minutes
```

Comment:
*"Terminate the installation"* sounds better to me, but native speakers should probably tell us what is the right wording here.